scm-plugins/scm-hg-plugin/src/main/resources/locales/en/plugins.json

```
@@ -23,7 +23,8 @@
       "disabledHelpText": "Enable or disable the Mercurial plugin.",
       "required": "This configuration value is required",
       "submit": "Submit",
-      "success": "Configuration changed successfully"
+      "success": "Configuration changed successfully",
+      "autoConfigure": "Configure mercurial automatically"
```

```suggestion
      "autoConfigure": "Configure Mercurial Automatically"
```
Button texts are treated like headings, see #1478.